quicknum & ItemUtil.setContainerItem fix

TRMMax opened the pull request:
quicknum now works in all inventories, and ItemUtil.setContainerItem now works in all slots.

KabanFriends commented:
Command registeration order may have been changed in the fabric client command API
Some commands like /color shows up the server side one

The way how commands are registered has been changed in API v2 so that's probably why. Not sure if there's a way to get around this

KabanFriends commented:
Here's a test build if you want to try out recode for 1.19:
[recode-mc1.19-unofficial-v3.zip](https://github.com/homchom/recode/files/9126197/recode-mc1.19-unofficial-v3.zip)

homchom commented:
@KabanFriends dev branch is no longer in use; please PR again to main.

TRMMax committed:
quicknum & ItemUtil.setContainerItem fix

quicknum now works in all inventories, and ItemUtil.setContainerItem now works in all slots.

BlazeMCworld reviewed (commented):
for me when i start using this and open any menu i crash with a stackoverflow error which loops at
`net.minecraft.client.gui.screens.inventory.AbstractContainerScreen.isHovering`
